packages/@okta/vuepress-site/docs/guides/mfa/activate-factor/sms/activatefactor.md

```diff
@@ -0,0 +1,4 @@
+1. Switch to Postman and from the **Factor Lifecycle Operations** folder, select the **Activate SMS Factor** request template.
```

Comment:
Switch to Postman and, in the **Factor Lifecycle Operations** folder, select the **Activate SMS Factor** request template.